spec/ios_update_metadata_source_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+require 'shared_examples_for_update_metadata_source_action'
+
+describe Fastlane::Actions::IosUpdateMetadataSourceAction do
+  before do
+    # This works around the `ensure_git_status_clean` call within the action.
+    # It would be quite cumbersome to iterate on the code if we had to commit
+    # every change _before_ running the tests...
```

Comment:
@AliSoftware, I wrote this comment before our discussion on soft vs hard wrapping. Once that's concluded, I'll update here, like with semantic line-breaks given the second line would still be "short".